Program.cs

```diff
@@ -1,2 +1,22 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using AnimalSound;
+
+namespace AnimalSound
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            List<Animal> animals = new List<Animal>
+            {
+                new Animal("Dog", "Bark"),
```

Comment:
new Animal("Batoza", "VRUM VRUM"), din nou, unde este incapsularea? de fapt in clasa Animal pot pune orice.